Source/ProgressionSystemRuntime/Public/Widgets/PSEndGameWidget.h

```diff
@@ -4,13 +4,13 @@
 
 #include "Blueprint/UserWidget.h"
 #include "PoolManagerTypes.h"
-#include "PSMenuWidget.generated.h"
+#include "PSEndGameWidget.generated.h"
 
 /**
  * Widget to display the progression as stars in the main menu.
  */
 UCLASS()
-class PROGRESSIONSYSTEMRUNTIME_API UPSMenuWidget : public UUserWidget
+class PROGRESSIONSYSTEMRUNTIME_API UPSEndGameWidget : public UUserWidget
```

Comment:
Please fix PS End-Game Widget warnings:
![image](https://github.com/user-attachments/assets/8a86a406-0a94-4625-b8f3-4689222ac741)

Reply:
done https://github.com/h4rdmol/ProgressionSystem/pull/43/commits/36201bc310628b43fe1a6a557348a3c4b2646006

Reply:
@h4rdmol, is not finished yet, only 2 warnings were fixed and 4 still stay in header:

- line 44 requires `enum class ECurrentGameState : uint8;` on the beginning of the  `PSEndGameWidget.h` (before UCLASS), search in bomber examples
- line 52 requires `enum class EEndGameState : uint8;` on the beginning of the  `PSEndGameWidget.h` (before UCLASS), , search in bomber examples
- line 48 requires `class`-forward right before `AMyPlayerState` usage

![image](https://github.com/user-attachments/assets/2e6c5bae-92e5-4299-ab9b-41cbff997885)

Reply:
fixed https://github.com/h4rdmol/ProgressionSystem/pull/43/commits/2f721df80cc5d382704d0a3a257c198ba8b71824